test-harness/Makefile

```diff
@@ -22,7 +22,7 @@ test:
 	retry --max=10 -- tango_admin --ping-device ska/telstate/01
 
 	cd /app && python setup.py test | tee setup_py_test.stdout
-	-cd /app && ./code-analysis.sh | tee code_analysis.stdout
+	cd /app && ./code-analysis.sh | tee code_analysis.stdout
```

Comment:
Are you sure of this? Does the pipeline work?

Reply:
Yes. The pipeline works.